bash/.bashrc.d/prompt

```diff
@@ -12,23 +14,23 @@ RESET=$(tput sgr0)
 YELLOW=$(tput setaf 3)
 
 LINECHAR="─"
-PREFIX="❱" # U+2771
+PREFIX_CHAR="❱" # U+2771
```

Comment:
Style thang here, but `LINECHAR` should probably now be renamed to `LINE_CHAR` to match `PREFIX_CHAR` and the rest of the variables in here.

Reply:
Resolved in https://github.com/cisagov/.dotfiles/pull/2/commits/069150a9663c4302f2804952169be9bc2e2c2fe6